### PR TITLE
Score of tweets is nil

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

**Comment by davgur** [comment]
[Score of Tweets is nil when sorting is not by relevance](https://trello.com/c/Qdu8645K/8-score-of-tweets-is-nil-when-sorting-is-not-by-relevance)

**Comment by YuriHechter** [comment]
Can we see the tweets if the search is ordered by date?
Also set the default score to 0.

File: search/tweets_engine.go [hunk 1]
```diff
@@ -45,6 +45,14 @@ func (e *ESEngine) SearchTweets(query Query, sortBy string, from int, size int,
 			err := errors.New(fmt.Sprintf("Failed tweets multi get: %+v", currentResults.Error))
 			return nil, err
 		}
+
+		for _, h := range currentResults.Hits.Hits {
+			if h.Score == nil {
+				s := float64(1)
```
Comment (YuriHechter) [inline search/tweets_engine.go]: Set default score to 0, not 1

File: search/tweets_engine.go [hunk 2]
```diff
@@ -45,6 +45,14 @@ func (e *ESEngine) SearchTweets(query Query, sortBy string, from int, size int,
 			err := errors.New(fmt.Sprintf("Failed tweets multi get: %+v", currentResults.Error))
 			return nil, err
 		}
+
+		for _, h := range currentResults.Hits.Hits {
+			if h.Score == nil {
```
Comment (YuriHechter) [inline search/tweets_engine.go]: Put this under the existed 'for' statement in line 78

**Pull request closed without merging**